tests/test_supabase.py

```
@@ -1,6 +1,6 @@
-from ..backend.app.model.supabase.models import User, Interaction, CodeChunk, Repository
+from backend.app.models.database.supabase import User, Interaction, CodeChunk, Repository
```

Pylint (3.3.7)

[error] No name 'CodeChunk' in module 'backend.app.models.database.supabase'

(E0611)